src/UI/Checkbox.elm

```diff
@@ -24,11 +24,6 @@ module UI.Checkbox exposing
 @docs withLabelVisible
 
 
-# Size
```

Comment:
Small docs fix here.
Maybe I should add docs to CI?

Reply:
@PedroHLC  that's a really good point, I will create a card for that